### ISSUE TITLE
Overflow should be auto instead of scroll?

### ISSUE THREAD

**Issue by mbenedettini** [react-component/table#367]
Not sure if this is an issue or intended behaviour. 

If I create a table setting scroll X to `true` I would expect the horizontal scrollbar to appear only if needed (in small screen sizes for example). That's achieved via `overflow-x: auto` but instead `overflow-x: scroll` is added making the scrollbar appear at all times even if the table width does not require scrolling.

https://codesandbox.io/s/keen-turing-nq853

**Comment by rtek** [comment]
Agreed. Work around is to `bodyStyle={{overflowX: auto}} scroll={{x: true}}`

Setting `scroll={{x: 'auto'}}` does not work but would be the ideal syntax.

**Comment by DeadManPoe** [comment]
Agreed. `overflow-x: scroll` makes visible the placeholder of the horizontal scrollbar, which is not ideal.

**Comment by nathan818fr** [comment]
Some news?

Setting overflowX to `'auto'` at https://github.com/react-component/table/blob/26497e0ad5e76e132707b781c35b6c0315e75b12/src/Table.tsx#L390 is working and seems a better behavior.

Like suggested by @rtek, `scroll={{x: 'auto'}}` would be a good syntax to do that.
